hack/release/pkg/bazel/flags.go

```diff
@@ -86,7 +86,10 @@ func (g *Bazel) Build(ctx context.Context, targets ...string) *exec.Cmd {
 // Bazel's --platforms variable will be set automatically.
 func (g *Bazel) BuildPlatform(ctx context.Context, os, arch string, targets ...string) *exec.Cmd {
 	platform := fmt.Sprintf("--platforms=@io_bazel_rules_go//go/toolchain:%s_%s", os, arch)
-	return g.Cmd(ctx, append([]string{"build", platform}, targets...)...)
+	// Set --stamp=true when running a build to workaround issues introduced
+	// in bazelbuild/rules_go#2110. For more information, see: https://github.com/bazelbuild/rules_go/pull/2110#issuecomment-508713878
+	// We should be able to remove the `--stamp=true` arg once this has been fixed!
+	return g.Cmd(ctx, append([]string{"build", platform, "--stamp=true"}, targets...)...)
```

Comment:
This fixes the issue introduced in bazelbuild/rules_go#2110 - hopefully we can get rid of the `--stamp=true` flag soon 😄